test/executor.test.js

```diff
@@ -373,6 +373,31 @@ describe('executor', function() {
     });
   });
 
+  describe('with boot script returning a promise and calling callback',
+    function() {
+      before(function() {
+        process.promiseAndCallback = true;
+      });
+
+      after(function() {
+        delete process.promiseAndCallback;
+      });
+
+      it('should only call the callback once',
+      function(done) {
+        var count = 0;
+        boot.execute(app, simpleAppInstructions(), function(err) {
+          if (err) return done(err);
+          count++;
+          expect(count).to.eql(1);
+          process.nextTick(function() {
+            done();
```

Comment:
I think this is rather brittle as it relies on `boot.execute` to make the second call of the callback before `process.nextTick` is run.

Fortunately, mocha detects the situation when the `done` callback was called twice and fails the test with a helpful error: `Error: done() called multiple times`.

I am proposing to simplify this test to simply pass `done` as the callback of `boot.execute`.